.github/workflows/cffconvert.yml

```diff
@@ -0,0 +1,65 @@
+################################# MIT LICENSE ##################################
+#
+# MIT License
+#
+# Copyright (c) 2022 Pablo Hamacher
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+################################################################################
```

Comment:
Why are you adding the License here again? It is already referenced down below and there is also just the normal `LICENSE` file in the root directory, as well as github showing the license of the project?

Reply:
I am usually using the GPL 2.0 license which asks the users to paste a certain header at the beginning of each source file.  Hence, it is something like a ritual for me to add such a license header to the source files.  I even add them to my `.gitignore`.  So, basically a file without such a header feels incomplete to me somehow.

The sole file I do not edited is `.gitmodules` since this one is maintained by Git and I do not want unintended side effects to take place.

After all, it is just a matter of taste but in my opinion, that way, users always see what rights they have to share and / or edit the corresponding source file.

Reply:
Yeah, I think that will just clutter the files too much.

I'm generally not too big of a fan of GPL 2.0, tbh. At the end it is always the responsibility of the user to check for the license.

So yeah, I do not want the license in every file 😅

Reply:
Done.